app/src/utils/utils.ts

```diff
@@ -59,6 +59,11 @@ export function isValidUrl(val: string) {
   return (val.includes('http') || val.includes('https')) && val.includes('://') && val.includes('.');
 }
 
+// Returns true if the provided address is valid (TODO support ENS)
+export function isValidAddress(val: string | undefined) {
+  return val && isAddress(val);
+}
```

Comment:
Is the purpose of this wrapper around `isAddress` just to allow us to pass in values which may be undefined? If so let's just add in a comment since it seems like an unnecessary wrapper method otherwise

Reply:
is this new? I don't think I made a change here, will change it to what it was before